content/en/docs/reference/config/analysis/ist0104/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: GatewayPortNotOnWorkload
+layout: analysis-message
+---
+
+This message occurs when you have a gateway that uses a specific port and refers
```

Comment:
I don't quite understand what this means, can we give an example maybe?

Reply:
fixed